backend/.claude/skills/impl/SKILL.md

```
@@ -0,0 +1,97 @@
+---
+name: impl
+description: 새 기능을 도메인 → 서비스 → 컨트롤러 순서로 TDD 사이클로 구현한다. 각 계층 완료 후 /write-tests를 호출하고, 전체 완료 후 code-reviewer를 실행한다.
+argument-hint: "[기능명] [--from=domain|service|controller]"
+allowed-tools: Read, Glob, Grep, Write, Edit, Bash, Agent
+---
+
+`$ARGUMENTS`에서 기능명과 시작 단계(`--from=`)를 파싱한다. `--from`이 없으면 `domain`부터 시작한다.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)